barretenberg/cpp/src/barretenberg/sumcheck/instance/prover_instance.hpp

```diff
@@ -64,6 +64,10 @@ template <class Flavor> class ProverInstance_ {
 
     ProverInstance_(Circuit& circuit)
     {
+        BB_OP_COUNT_TIME_NAME("ProverInstance(Circuit&)");
```

Comment:
Without the composer, we need place for this logic. Don't want to always have to explicitly finalize etc the builder everywhere. Makes sense to me that it goes here.